test_scripts/API/VehicleData/HandsOffSteering/common.lua

```diff
@@ -0,0 +1,309 @@
+          hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+          parameters = { "handsOffSteering" }
+        },
+        OnVehicleData = {
+          hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+          parameters = { "handsOffSteering" }
+        },
+        SubscribeVehicleData = {
+          hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+          parameters = { "handsOffSteering" }
+        },
+        UnsubscribeVehicleData = {
+          hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+          parameters = { "handsOffSteering" }
+        }
+      }
+    }
+  end
+  pt.policy_table.functional_groupings["NewTestCaseGroup"] = pGroup
+  pt.policy_table.app_policies["default"].groups = { "Base-4", "NewTestCaseGroup" }
+  pt.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+  m.setPreloadedPT(pt)
+end
+
+--[[ @preconditions: Clean environment, backup and update of sdl_preloaded_pt.json file
+ --! @parameters:
+ --! pGroup: data for updating sdl_preloaded_pt.json file
+ --! @return: none
+ --]]
+function m.preconditions(pGroup)
+  actions.preconditions()
+  m.updatedPreloadedPTFile(pGroup)
+end
+
+--[[ @setHashId: Set hashId value which is required during resumption
+--! @parameters:
+--! pHashId: hashId value to store
+--! pAppId: application number (1, 2, etc.)
+--! @return: none
+--]]
+function m.setHashId(pHashId, pAppId)
+  hashId[pAppId] = pHashId
+end
+
+--[[ @getHashId: Get hashId value of an app which is required during resumption
+--! @parameters:
+--! pAppId: application number (1, 2, etc.)
+--! @return: app's hashId
+--]]
+function m.getHashId(pAppId)
+  return hashId[pAppId]
+end
+
+--[[ @getVehicleData: Processing GetVehicleData RPC
+--! @parameters:
+--! pHandsOffSteering: handsOffSteering parameter value
+--! pAppId: application number (1, 2, etc.)
+--! @return: none
+--]]
+function m.getVehicleData(pHandsOffSteering, pAppId)
+  if not pAppId then pAppId = 1 end
+  local cid = m.getMobileSession(pAppId):SendRPC("GetVehicleData", { handsOffSteering = true })
+  m.getHMIConnection():ExpectRequest("VehicleInfo.GetVehicleData", { handsOffSteering = true })
+  :Do(function(_, data)
+    m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { handsOffSteering = pHandsOffSteering })
+  end)
+  m.getMobileSession(pAppId):ExpectResponse(cid,
+    { success = true, resultCode = "SUCCESS", handsOffSteering = pHandsOffSteering })
+end
+
+--[[ @processSubscriptionRPCsSuccess: Successful processing Subscribe/Unsubscribe RPC for handsOffSteering data
+--! @parameters:
+--! pRpcName: RPC name
+--! pAppId: application number (1, 2, etc.)
+--! isRequestOnHMIExpected: true - in case VehicleInfo.Sub/UnsubscribeVehicleData request on HMI is expected,
+--! otherwise - false
+--! pResultCode: resultCode value for expectation on App
+--! @return: none
+--]]
+function m.processSubscriptionRPCsSuccess(pRpcName, pAppId, isRequestOnHMIExpected, pResultCode)
```

Comment:
I found the usage of `pResultCode` only in one script and besides common file has the `processRPCUnsuccessRequest` function for the processing unsuccess cases, so please move the logic for `pResultCode` to `processRPCUnsuccessRequest` function. This function is about `Successful processing Subscribe/Unsubscribe RPC`